CONTRIBUTING.md

```diff
@@ -29,6 +29,25 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for contributing!
 
+### Local setup
+
+```sh
+make deploy-full-local-setup
```

Comment:
Maybe we can rename this target to simply become `deploy` for the sake of simplicity of first-user experience?
iirc we don't have one-cluster setup targets anyway, and this one acts as default deployment in most cases.

Reply:
deploy here means composite of 2 clusters+ k8gb helm chart.+ CR + test apps. Just `deploy` is semantically more about k8gb helm chart only. That's why we have so many different `deploy-*` targets. To handle different levels of granularity

Reply:
It's hard to see mentioned target semantics unless you thoroughly investigate Makefile. 
Deploy is just deploy to me and can be done in different ways (helm, kustomize...), but I don't see this in target names. Also most of `deploy-*` targets we use are deploying to 2 clusters, but I don't see mentioning of that as well, also they all deploy to local setup:
- deploy-full-local-setup
- deploy-stable
- deploy-candidate
- upgrade-candidate
Hence I suggest renaming of `deploy-full-local-setup` to simply `deploy`, because there's no sense of using `full` and `local-setup` there if we want to be consistent and simple.

Reply:
@somaritane I suggest we create another Issue for renaming-based refactoring if needed. This PR mainly targets the current degraded state of `deploy-full-local-setup` and unreliable state of `upgrade-testing` pipeline.

Reply:
@ytsarev Sure, let's split